parser-typechecker/src/Unison/Codebase/SqliteCodebase/Migrations/MigrateSchema16To17.hs

```diff
@@ -89,6 +91,19 @@ migrateSchema16To17 conn = withDisabledForeignKeys $ do
       let enable = Connection.execute conn [Sqlite.sql| PRAGMA foreign_keys=ON |]
       let action = Sqlite.runWriteTransaction conn \run -> run $ m
       UnsafeIO.bracket disable (const enable) (const action)
+    initializeCurrentProjectPath :: ProjectId -> ProjectBranchId -> [NameSegment] -> Sqlite.Transaction ()
+    initializeCurrentProjectPath projId branchId path = do
+      Sqlite.execute
+        [Sqlite.sql| DELETE FROM current_project_path |]
+      Sqlite.execute
+        [Sqlite.sql|
+          INSERT INTO current_project_path(project_id, branch_id, path)
+          VALUES (:projId, :branchId, :jsonPath)
+        |]
+      where
+        jsonPath :: Text
+        jsonPath =
+          Text.Lazy.toStrict (Aeson.encodeToLazyText $ NameSegment.toUnescapedText <$> path)
```

Comment:
Froze this code via copy-pasta since it's part of a migration and shouldn't change.